Source/Resources/Content/MSGraph/Groups/BUILTIN - Device Enrollers.json

```diff
@@ -1,6 +1,6 @@
 {
   "assignedLicenses": [],
-  "description": "This group is used to identify users that may enroll a non-Autopilot Windows device into Intune MDM. If a user is not a member of this group, the device must first be registered for Autopilot in order for the user to enroll.",
+  "description": "What is it: A manual group used to determine users that have permissions to enroll a device that is not registered with Autopilot. \n Simeon Notes: This group will be required to enroll new devices into your environment that have not been pre-registered by uploading of the device hardware hash.",
```

Comment:
**What does this do?**
Creates a group of users that have permission to enroll a device that is not registered with Autopilot. If a user is not a member of this group, they cannot enroll a non-autopilot device.

**Why should you use this?**
If you want less restrictions for certain users to be able to enroll new devices into your environment that have not been pre-registered by uploading the device hardware hash.

**What is the end user impact?**
Certain users will be able to have devices enrolled that were not Autopiloted.

**Learn more**
NA